src/test/java/com/spring/boot/ApplicationTests.java

```diff
@@ -20,7 +28,21 @@ public void readCorrect() {
     }
 
     @Test
-    void contextLoads() {
```

Comment:
@Bogdan-Demianchuk Why you remove the existing test method?

Reply:
Accidentally